editor/js/Menubar.js

```diff
@@ -17,10 +17,10 @@ function Menubar( editor ) {
 	container.add( new MenubarFile( editor ) );
 	container.add( new MenubarEdit( editor ) );
 	container.add( new MenubarAdd( editor ) );
-	//	container.add( new MenubarPlay( editor ) );
-	//	container.add( new MenubarExamples( editor ) );
-	//	container.add( new MenubarView( editor ) );
-	//	container.add( new MenubarHelp( editor ) );
+	container.add( new MenubarPlay( editor ) );
+	container.add( new MenubarExamples( editor ) );
+		// container.add( new MenubarView( editor ) );
```

Comment:
Let's not change the lines if we don't need to, pls, to avoid conflicts in the future

Reply:
I guess it's because of eslint